randovania/games/am2r/exporter/patch_data_factory.py

```diff
@@ -110,18 +115,18 @@ def get_locked_ammo_text(ammo_item: str):
         pb_text = get_locked_ammo_text("Power Bomb Expansion")
 
         game_patches = {
-            "septogg_helpers": self.patches.configuration.septogg_helpers,
-            "respawn_bomb_blocks": self.patches.configuration.respawn_bomb_blocks,
-            "skip_cutscenes": self.patches.configuration.skip_cutscenes,
-            "skip_item_cutscenes": self.patches.configuration.skip_item_cutscenes,
-            "energy_per_tank": self.patches.configuration.energy_per_tank,
-            "grave_grotto_blocks": self.patches.configuration.grave_grotto_blocks,
-            "fusion_mode": self.patches.configuration.fusion_mode,
-            "supers_on_missile_doors": self.patches.configuration.supers_on_missile_doors,
-            "nest_pipes": self.patches.configuration.nest_pipes,
-            "softlock_prevention_blocks": self.patches.configuration.softlock_prevention_blocks,
-            "a3_entrance_blocks": self.patches.configuration.a3_entrance_blocks,
-            "screw_blocks": self.patches.configuration.screw_blocks,
+            "septogg_helpers": config.septogg_helpers,
+            "respawn_bomb_blocks": config.respawn_bomb_blocks,
+            "skip_cutscenes": config.skip_cutscenes,
+            "skip_item_cutscenes": config.skip_item_cutscenes,
```

Comment:
Afaik the `skip_save_cutscene` needs to be merged here now.

Reply:
Fixed via merging